libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -25,14 +24,25 @@ export function Dialog({
     'fixed w-[520px] px-28 py-24 top-[50%] left-[50%] translate-x-[-50%] translate-y-[-50%]',
     // Need to apply background and text colors again as content is rendered in a portal
     'dark:bg-black dark:text-white-95 bg-white text-black-95',
-    getIntentShadow(intent)
+    'shadow-callout',
```

Comment:
minor (non-blocking): I managed to get utility functions to work in BE. We can still use this if you prefer, you can check out #187 for the eventual implementation. I think the key was getting rid of the classnames function IIRC